main/schemas/board.json

```diff
@@ -0,0 +1,226 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$ref": "#/definitions/Welcome9",
+    "definitions": {
+        "Welcome9": {
+            "type": "object",
+            "additionalProperties": true,
+            "properties": {
+                "id": {
+                    "type": "string"
+                },
+                "name": {
+                    "type": "string"
+                },
+                "desc": {
+                    "type": "string"
+                },
+                "descData": {
+                    "type": "null"
+                },
+                "closed": {
+                    "type": "boolean"
+                },
+                "idOrganization": {
+                    "type": "string"
+                },
+                "idEnterprise": {
+                    "type": "null"
```

Comment:
"idEnterprise" is set as "required" but also set as "type":"null".
This will generate a problem when the response actually returns and id for the enterprise.
(setting "type" as {} will allow any type of data, useful for when you don't know the type)

Reply:
I just realized this existed, i'll fixed for tomorrow